config/src/config.rs

```diff
@@ -148,6 +148,9 @@ pub struct Config {
     #[dynamic(default = "default_command_palette_font_size")]
     pub command_palette_font_size: f64,
 
+    #[dynamic(default = "default_command_palette_rows")]
+    pub command_palette_rows: usize,
+
```

Comment:
Let's make this optional:

```suggestion
    pub command_palette_rows: Option<usize>,
```